### PR TITLE
maintain v18 version

### PR DESCRIPTION
**Pull request by bgruening**
I would like to keep a v18 build and rebuild for newer python versions if possible.

Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by ocefpaf** [comment]
@conda-forge-admin please rerender

File: conda-forge.yml [hunk 1]
```diff
@@ -1,3 +1,5 @@
+bot:
+  abi_migration_branches: [v18]
```
Comment (ocefpaf) [inline conda-forge.yml:2]: @bgruening this will ensure that the bot keeps sending migration PRs to this branch.

Comment (bgruening) [reply conda-forge.yml:2]: Oh that is cool! Thanks!

**Comment by bgruening** [comment]
Should we just skip the ARM and Win builds?

**Comment by ocefpaf** [comment]
> Should we just skip the ARM and Win builds?

If you don't care about this archs here I would just merge it without skipping.

**Comment by bgruening** [comment]
I do not care about them. A user only needs the Linux builds. Thanks!

**Pull request merged**